frontend/src/service-panel/VpInfo.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+
+import * as classNames from 'classnames';
+import config from '../config';
+
+interface VpInfoProps {
+    isVisible: boolean;
+}
+
+const VpInfo: React.StatelessComponent<VpInfoProps> = ({ isVisible }) => {
+  const systemInfoUrl = config.api.adminUrl + '/admin.php?page=versionpress/admin/system-info.php';
+  const wrapperClassName = classNames({
+    'ServicePanel-wrapper': true,
+    'ServicePanel-wrapper--hide': !isVisible,
+  });
+
+  return (
```

Comment:
This could be also extracted to 2 components. Maybe something like `panel/warning`, `panel/support`?